proxy/src/main/java/com/wavefront/agent/ChannelByteArrayHandler.java

```diff
@@ -63,9 +56,17 @@ protected void channelRead0(ChannelHandlerContext ctx, byte[] msg) throws Except
     try {
       decoder.decodeReportPoints(msg, points, "dummy");
       for (final ReportPoint point: points) {
-        if (!this.metricPredicate.apply(point.getMetric())) {
-          pointHandler.handleBlockedPoint(point.getMetric());
-          continue;
+        if (preprocessor != null) {
+          // backwards compatibility: apply "pointLine" rules to metric name
+          if (!preprocessor.forPointLine().filter(point.getMetric())) {
+            pointHandler.handleBlockedPoint(preprocessor.forReportPoint().getLastFilterResult());
+            continue;
+          }
+          preprocessor.forReportPoint().transform(point);
```

Comment:
This, and https://github.com/wavefrontHQ/java/pull/91/files/b1a67233b867796c5177ea9c5c8c03bfba610f59#diff-218e7fd1a369e3a4a715cc0ee4405ed4L90